### DISCUSSION TITLE
Lesson 7: I don't know why I get an error when I use `assertEq` and `import "forge-std/Test.sol";` in my test file!

### DISCUSSION THREAD

**Discussion by IAm0xAxe** [Cyfrin/foundry-full-course-cu#281] [Q&A] [answered]
Lesson 7: I don't know why I get an error when I use `assertEq` and `import "forge-std/Test.sol";` in my test file! I'm following all the steps the instructor told me to do!

**Me:** 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/d2abc31e-280c-4f34-9e50-cfe6558bbf05)

**Teacher:** 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33568293/9be2dc15-b747-4905-ac25-58134d2d6042)

**Comment by IAm0xAxe** [comment, marked as answer]
Okay,My bad. 

1. I didn't write `is Test`  https://github.com/foundry-rs/foundry/issues/4993

2. Use the `forge remappings > remappings.txt` command to remap, and finally restart VS code again. `import "forge-std/Test.sol";` will not report an error!

**Comment by JacobHomanics** [comment]
Came here with the same problem and followed Patrick's advice by going to the discussions! And voila! Someone else already encountered the same issue as me and was able to find the answer! Thanks for sharing @IAm0xAxe!

**Reply by IAm0xAxe** [reply]
You're welcome
